ai_genomics/pipeline/samples/make_samples.py

```diff
@@ -0,0 +1,91 @@
+import pandas as pd
+from ai_genomics import bucket_name
+from ai_genomics.getters.crunchbase import get_ai_genomics_crunchbase_org_ids
+from ai_genomics.getters.data_getters import save_to_s3
+from ai_genomics.getters.gtr import get_ai_genomics_gtr_data
+from ai_genomics.getters.openalex import get_openalex_ai_genomics_works, work_abstracts
+from ai_genomics.getters.patents import get_ai_genomics_patents
+from ai_genomics.utils.crunchbase import fetch_crunchbase, parse_s3_table
+
+YEARS = range(2007, 2022)
+SAMPLE_SIZE = 100
+
+
+def openalex_abstracts_dict_to_df(openalex_abstracts: dict) -> pd.DataFrame:
+    """Converts OpenAlex abstracts into a DataFrame"""
+    return (
+        pd.DataFrame.from_dict(openalex_abstracts, orient="index")
+        .reset_index()
+        .rename(columns={"index": "work_id", 0: "abstract_text"})
+    )
+
+
+def make_ai_genomics_openalex_samples(sample_size: int = SAMPLE_SIZE) -> pd.DataFrame:
```

Comment:
I was checking these out and because the open alex tagger pulls open alex tags, jack v didn't use the open alex tagger on this sample. Is there anyway you can output the open alex tags just for this dataset?